Tests/web3swiftTests/localTests/BasicLocalNodeTests.swift

```diff
@@ -64,12 +65,12 @@ class BasicLocalNodeTests: LocalTestCase {
         print("Balance before to: " + balanceBeforeTo.description)
         print("Balance before from: " + balanceBeforeFrom.description)
 
-        let result = try await sendTx.writeToChain(password: "web3swift")
-        let txHash = result.hash
+        let result = try! await sendTx.writeToChain(password: "web3swift")
```

Comment:
Force try here's unnecessary, despite that I doubt that it's a cause of the broken tests, seems like if we remove it they just keep failing all the same way.

Reply:
Agree, remove the exclamation mark.
Don't recall why I've added it as it doesn't make sense here.